runtime/src/main/scala/akka/grpc/javadsl/GrpcExceptionHandler.scala

```diff
@@ -4,48 +4,56 @@
 
 package akka.grpc.javadsl
 
+import java.lang.{ Iterable => jIterable }
 import java.util.concurrent.CompletionException
+import java.util.{ ArrayList => jArrayList }
```

Comment:
(I think this can also be just `ArrayList`)

Reply:
Done!